src/cart/crt_iv.c

```diff
@@ -20,6 +20,7 @@
 
 #include "crt_internal.h"
 #include <cart/iv.h>
+#include <daos/common.h>
```

Comment:
this was left in from another attempt to enhanced the new log messages with a bit of a hack to reveal the pool UUID. Needs to be removed, since that effort was abandoned.